static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -61,20 +61,19 @@ const monthlySupporterCost = (location: string): string => {
     return payment || '£5';
 };
 
-const supporterEngagementBannerCopy = (location: string): string =>
-    engagementBannerCopy(
-        `Support us for ${monthlySupporterCost(location)} per month.`
-    );
+const supporterEngagementCtaCopy = (location: string): string =>
+    `Support us for ${monthlySupporterCost(location)} per month.`;
 
 const contributionEngagementBannerCopy = (): string =>
-    engagementBannerCopy(`Support us with a one-time contribution`);
+    `Support us with a one-time contribution`;
 
 const supporterParams = (location: string): EngagementBannerParams =>
     Object.assign({}, baseParams, {
         buttonCaption: 'Become a Supporter',
         linkUrl: 'https://membership.theguardian.com/supporter',
         products: ['MEMBERSHIP_SUPPORTER'],
-        messageText: supporterEngagementBannerCopy(location),
+        messageText: engagementBannerCopy(),
+        ctaText: supporterEngagementCtaCopy(location),
         pageviewId: (config.ophan && config.ophan.pageViewId) || 'not_found',
```

Comment:
Optional: this assignment can be replaced with:
```js
pageviewId: config.get('ophan.pageViewId', 'not_found'),
```
Same on line 87

Reply:
nice!